src/lib/flocking/ugens/envelopes.js

```diff
@@ -0,0 +1,1101 @@
+/*
```

Comment:
Cool - in the meantime, can we get both flocking and infusion out into npm references to get them out of this diff? I assume that we can generate the infusion build using an npm "postinstall" step.